pants.toml

```diff
@@ -50,6 +55,15 @@ args = [
 # Resolve-global `scalac` plugins, which are defined as `scalac_plugin` targets.
 jvm-default = "acyclic"
 
+[scala-infer]
```

Comment:
Note: The deprecation warning states this section is `scala_infer` (underscore) when in fact it is a `-` dash.